### DISCUSSION TITLE
Request for relevant code sourecs accompanying "Progressive Seed Generation Auto-Encoder for Unsupervised Point Cloud Learning"

### DISCUSSION THREAD

**Discussion by BBBOYGIRL** [community/community#130449] [Programming Help]
### Body

I am currently working on a project related to unsupervised learning for point cloud data and came across the paper titled "Progressive Seed Generation Auto-Encoder for Unsupervised Point Cloud Learning." I find the concepts introduced in the paper particularly intriguing and believe they could significantly enhance my project.

However, I am having difficulty understanding some of the implementation details that were not fully covered in the paper. Specifically, I am struggling with the seed generation process and how it integrates with the auto-encoder architecture for effective feature learning.

I would greatly appreciate it if someone could share the source codes from the actual implementation. Any additional resources or explanations that could help clarify the procedural steps or architectural decisions would also be incredibly helpful.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by tanyatiwari995** [comment]
How to Fix This Problem:
To resolve this, follow these steps:

1. Search for Official Code Repository
Many research papers have accompanying GitHub repositories or supplementary materials.
Search for the paper title along with keywords like "GitHub", "implementation", or "code".
Check the paper’s official website, conference page, or arXiv/Google Scholar for any provided links.
2. Look for Similar Implementations
If no official code is available, look for similar Point Cloud Auto-Encoder implementations.
Try searching GitHub with:

Progressive Seed Generation Auto-Encoder Point Cloud Learning
or

Unsupervised Point Cloud Autoencoder GitHub
Some repositories might not directly reference the paper but implement similar concepts.
3. Understand the Seed Generation Process
If documentation is unclear, break the seed generation process into key steps:
How are seeds initialized?
How do they evolve in the network?
What loss functions or constraints apply to them?
Look for open-source implementations of auto-encoders in point cloud learning and study their approach.
4. Ask in Relevant Research Forums
Try reaching out to the authors via email (often listed in the paper).
Post a question on:
Reddit (r/MachineLearning)
ResearchGate
AI/ML GitHub discussions
5. Check for Code in Related Papers
Often, research is built upon previous work. Look for papers that cite this paper (Google Scholar → "Cited By").
They may provide a clearer implementation or code references.
